### PR TITLE
Add a "Health Check" endpoint for readiness probes to call on OpenShift

### PR DESCRIPTION
**Pull request by jguertin**

### CODE REVIEW

**Review by bryan-gilbert — COMMENTED** [review]
nice and simple

**Review by bryan-gilbert — APPROVED** [review]

File: ppr-api/src/endpoints/healthcheck.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from fastapi import APIRouter
+
+router = APIRouter()
```
Comment (WalterMoar) [inline ppr-api/src/endpoints/healthcheck.py]: I'd like to go with more explicit calls to module functions:

```python
import fastapi

router = fastapi.APIRouter()
```

File: ppr-api/src/main.py [hunk 2]
```diff
@@ -2,7 +2,10 @@
 from fastapi import FastAPI, Header, HTTPException
 from starlette.middleware.cors import CORSMiddleware
 
+from endpoints import healthcheck
+
 app = FastAPI()
+app.include_router(healthcheck.router)
```
Comment (WalterMoar) [inline ppr-api/src/main.py:8]: I'm going to change this when I fix my merge conflicts.

File: ppr-api/src/endpoints/healthcheck.py (hunk 1 above)
Comment (jguertin) [reply ppr-api/src/endpoints/healthcheck.py]: sure

File: ppr-api/src/endpoints/healthcheck.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+
+@router.get("/health", include_in_schema=False)
```
Comment (WalterMoar) [inline ppr-api/src/endpoints/healthcheck.py]: We should include this endpoint in the OpenAPI docs - we won't expose it through the gateway, but it saves digging in the code when we want to find the check.

When I fix my merge conflicts I'll separate it out in the docs.

**Review by WalterMoar — CHANGES_REQUESTED** [review]
One change with module conventions and readability, to always explicitly use the module name.

File: ppr-api/src/main.py (hunk 2 above)
Comment (jguertin) [reply ppr-api/src/main.py:8]: 👍

File: ppr-api/src/endpoints/healthcheck.py (hunk 3 above)
Comment (jguertin) [reply ppr-api/src/endpoints/healthcheck.py]: Sure, if you can separate it out from the docs that's great.  I've removed the flag.

**Review by WalterMoar — APPROVED** [review]

**Comment by WalterMoar** [comment]
Gonna merge this so I can grab it.

**Pull request merged**